tests/conftest.py

```diff
@@ -53,3 +53,9 @@ def pytest_terminal_summary(terminalreporter):
     make_reports = terminalreporter.config.getoption("--make-reports")
     if make_reports:
         pytest_terminal_summary_main(terminalreporter, id=make_reports)
+
+
+def pytest_sessionfinish(session, exitstatus):
```

Comment:
Do we need the same for `examples/conftest.py`?

Reply:
Not for now, since the example tests are run without filtering any tests. This is here since we can end up with one of the jobs running no tests (because the markers skipped them all for instance) but the examples job runs all examples tests (if it's launched).